presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -141,14 +141,17 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
     @Override
     public Optional<ColumnMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
     {
-        switch (typeHandle.getJdbcTypeName()) {
+        String jdbcTypeName = typeHandle.getJdbcTypeName()
+                .orElseThrow(() -> new PrestoException(JDBC_ERROR, "Type name is missing: " + typeHandle));
```

Comment:
nit

```suggestion
                // typeName is always present in case of PostgreSQL
                .orElseThrow(() -> new PrestoException(JDBC_ERROR, "Type name is missing: " + typeHandle));
```